test/framework/client/collect.go

```diff
@@ -126,8 +126,7 @@ func OutputFormat(format string) CollectResponsesOptsFn {
 		// parse the format they specified.
 		opts.Flags = append(opts.Flags,
 			"--silent",
-			"--output", os.DevNull,
-			"--write-out", opts.ShellEscaped(format),
+			"--write-out", format,
```

Comment:
Why you removed `ShellEscaped`?

Reply:
Fixed it by changing the ordering of the options